content/webapp/pages/guides/exhibitions/[id]/[type].tsx

```diff
@@ -266,14 +359,36 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
           ) : (
             <>
               {type !== 'captions-and-transcripts' && (
-                <p>This exhibition has {numberedStops.length} stops.</p>
+                <p>This exhibition has {numberOfStops} stops.</p>
               )}
             </>
           )}
         </Layout>
       </Space>
-
-      <ExhibitionGuideStops type={type} stops={exhibitionGuide.components} />
+      {/* For deprecated ExhibitionGuides */}
```

Comment:
Are we thinking of migrating those to the new ones/deleting them eventually?

Reply:
I'm not sure, I think it's more likely that the get deleted than migrated - I'll raise it with the content team